src/pages/team.en.tsx

```diff
@@ -1,188 +1,120 @@
-                    <SocialIcon
-                      key={i}
-                      url={link}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                      bgColor="#0096D7"
-                      style={{ height: 40, width: 40 }}
-                    />
-                  ))}
-              </div>
-              <p>{member.description.description}</p>
-            </div>
-          ))}
-        </div>
-      </section>
+type MemberCardInfo = {
+  name: string;
+  title: string;
+  description: {
+    description: string;
+  };
+  photo: {
+    fluid: {
+      src: string;
+    };
+  };
+};
 
-      <section className="hero is-small">
-        <div className="hero-body has-text-centered is-horizontal-center">
-          <div className="container">
-            <h1 className="title is-size-2 has-text-grey-dark has-text-weight-normal is-spaced">
-              {props.content.designAdvisorsTitle}
-            </h1>
-          </div>
-        </div>
-      </section>
+const MemberCard: React.FC<MemberCardInfo> = (props) => (
+  <div className="column is-3 is-paddingless">
+    <figure className="image">
+      <img
+        className="is-rounded"
+        src={props.photo.fluid.src}
+        alt={props.name}
+      />
+    </figure>
+    <div className="has-text-centered my-7">
+      <h3>{props.name}</h3>
+      <h4>{props.title}</h4>
+    </div>
+    <div>
+      <p>{props.description.description}</p>
+    </div>
+  </div>
+);
 
-      <section className="content-wrapper design-advisors">
-        <div className="columns is-multiline">
-          {props.content.designAdvisors.map((advisor: any, i: number) => (
-            <div
-              className="column team-member is-one-third has-text-centered "
-              key={i}
-            >
-              <figure className="image is-128x128 is-horizontal-center">
-                <img
-                  className="is-rounded"
-                  src={advisor.photo.file.url}
-                  alt={advisor.name}
-                />
-              </figure>
+type MemberCardsListInfo = {
+  sectionTitle: string;
+  memberList: MemberCardInfo[];
+};
 
-              <h4 className="title is-size-5 has-text-grey-dark">
-                {advisor.name}
-              </h4>
-              <h4 className="subtitle is-size-5 has-text-grey-dark">
-                {advisor.title}
-              </h4>
-              <p>{advisor.description.description}</p>
-            </div>
-          ))}
-        </div>
-      </section>
+const MemberCardsList: React.FC<MemberCardsListInfo> = (props) => (
+  <div className="py-9">
+    <div className="columns">
+      <div className="column is-12">
+        <h1 className="team-title">{props.sectionTitle}</h1>
+      </div>
+    </div>
+    {chunkArray(props.memberList, 3).map((members: Array<any>, i: number) => (
+      <div className="members columns py-9" key={i}>
```

Comment:
Love this approach! I'd add a comment somewhere explaining your approach here. At first glance, I was confused why you had to do this instead of just adding the `is-multiline` class to the columns div—but I can see now that you are trying to fit only 3 columns of 1/4 width onto one line instead of 4 and therefore need to add this "chunking."

Reply:
Thanks, talking through the nested column idea - that was super helpful, and I decided to go with that approach instead - I think it's a bit more simple and clear